libs/wordpress-plugin/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@blockprotocol/wordpress",
+  "version": "0.0.2",
+  "private": true,
+  "description": "Repository for developing the Block Protocol WordPress plugin",
+  "homepage": "https://blockprotocol.org/wordpress",
+  "bugs": {
+    "url": "https://github.com/blockprotocol/blockprotocol/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/blockprotocol/blockprotocol.git",
+    "directory": "libs/wordpress-plugin"
+  },
+  "license": "see LICENSE.md",
+  "author": {
+    "name": "HASH",
+    "url": "https://hash.ai"
+  },
+  "scripts": {
+    "build": "composer install -d plugin/trunk && wp-scripts build",
+    "dev:plugin": "wp-scripts start",
+    "dev:wordpress": "docker compose -f wordpress/docker-compose.yml  up",
+    "fix:eslint": "eslint --fix .",
+    "postinstall": "patch-package  --error-on-warn",
```

Comment:
I get the following error when running `yarn`:
```console
$ patch-package  --error-on-warn
patch-package 6.5.1
Applying patches...

**ERROR** Failed to apply patch for package @apidevtools/json-schema-ref-parser

  This happened because the patch file patches/@apidevtools+json-schema-ref-parser+9.1.2.patch could not be parsed.

  If you just upgraded patch-package, you can try running:

    patch -p1 -i patches/@apidevtools+json-schema-ref-parser+9.1.2.patch
    npx patch-package @apidevtools/json-schema-ref-parser

  Otherwise, try manually creating the patch file again.

  If the problem persists, please submit a bug report:

    https://github.com/ds300/patch-package/issues/new?title=Patch+file+parse+error&body=%3CPlease+attach+the+patch+file+in+question%3E
```

Reply:
Thank you – I edited the comment without changing the line numbers which messed it up. Fixed in https://github.com/blockprotocol/blockprotocol/pull/1123/commits/dee8820fab64668c58aee20642faecc83f982b01